venter/tests.py

```diff
@@ -201,3 +221,43 @@ def test_model_venter(self):
         )
 
         self.assertEqual(str(complaintMedia), 'www.google.com')
+
+    def test_admin_actions(self):
+        self.complaint_admin = ComplaintModelAdmin(Complaints, AdminSite())  # pylint: disable=W0201
+
+        def create_complaint(user, **kwargs):
+            Complaints.objects.create(created_by=user, **kwargs)
+
+        create_complaint(self.user.profile, status='Reported')
+        queryset = Complaints.objects.filter(status='Reported')
```

Comment:
You should refactor all the inline strings to somewhere, since a hard-to-see spelling error anywhere in the future can potentially wreck havoc. Then use the globally constant strings here, as well as in `admin.py`